pages/form/Thankyoupage.jsx

```diff
@@ -0,0 +1,21 @@
+import { Heading, Text, Box } from "@chakra-ui/react";
+import Layout from "../../components/Layout/Layout";
+
+export default function Question5() {
```

Comment:
Could you rename this function please?

Reply:
OOOPS... How did my eye miss that. Fixed. Pushing the changes now.